tracee/tracee.go

```diff
@@ -181,50 +181,56 @@ func (t *Tracee) initBPF(ebpfProgram string) error {
 
 	sysPrefix := bpf.GetSyscallPrefix()
 	for _, e := range eventsToTraceFinal {
-		if eName, isSyscall := EventsSyscalls[e]; isSyscall {
-			kp, err := t.bpfModule.LoadKprobe(fmt.Sprintf("syscall__%s", eName))
+		event, ok := EventsIDToEvent[e]
```

Comment:
instead of lookup each event by id, we can change eventsToTraceFinal from []int32 to []EventConfig

Reply:
Let's do in another commit